Retos/Reto #34 - EL TXT [Media]/python/KevinED11.py

```diff
@@ -120,7 +120,7 @@ def supported_writing_modes(self) -> list[FileManipulationModes]:
 class FileReaderAndWriter:
     reader: IReader
     file_writer: IFileWriter
-
+    
```

Comment:
Your PR title express that you will contribute on the challenge number 35, why do you edit this file?, can you contribute on this using a differente PR?